src/libs/relay/conduit_relay_io_silo.cpp

```diff
@@ -3857,32 +3848,32 @@ void silo_write_ucd_zonelist(DBfile *dbfile,
     }
 
     Node n_conn_compact;
-    detail::conditional_compact(n_mesh_conn, n_conn_compact);
+    detail::conditional_compact(n_conn, n_conn_compact);
 
     int conn_len = n_conn_compact.total_bytes_compact() / sizeof(int);
     int *conn_ptr = (int *)n_conn_compact.data_ptr();
 
     n_mesh_info[topo_name]["num_elems"].set(total_num_elems);
 
-    std::string zlist_name = topo_name + "_connectivity";
+    const std::string zlist_name = (write_overlink ? "zonelist" : topo_name + "_connectivity");
 
     int silo_error =
         DBPutZonelist2(dbfile,             // silo file
                        detail::sanitize_silo_varname(zlist_name).c_str(), // silo obj name
                        total_num_elems,    // number of elements
-                       2,                  // spatial dims
+                       2,                  // spatial dims - TODO ?????
```

Comment:
What do spatial dims mean in the context of a zonelist? Is this just the dimension of the shape type? Does it matter?

Reply:
I think it is the dimension of the underlying coordinate space.  
2d coord system would be 2 for rz, or xy, etc
3d coord system would be 3 for xyz

Reply:
Ok. I'll get that worked out.

I'm not sure if matters though - this has been hardcoded to 2 since before I worked on the silo i/o support. Changing it to 3 doesn't seem to have any effect that I can see. But probably a good call just in case downstream tools use the information.